### PR TITLE
rmp-serde support

### PR DESCRIPTION
**Pull request by lu-zero**
I quickly rebased the set and it seems to build at least.

### CODE REVIEW

File: src/codec/postcard_1_0.rs [hunk 1]
```diff
@@ -1,18 +1,59 @@
...
+/// ```rust
+/// #[derive(Clone, Default, serde::Deserialize, serde::Serialize)]
+=======
+/// ```no_run
+/// # fn main() {
+/// use serde::{Deserialize, Serialize};
+/// use native_model::native_model;
+///
+/// #[derive(Clone, Default, Deserialize, Serialize)]
+>>>>>>> bded1fc (Turn off doc tests)
```
Comment (vincent-herlemont) [inline src/codec/postcard_1_0.rs:34]: @lu-zero Can you remove these diff characters?

**Review by vincent-herlemont — COMMENTED** [review]
@lu-zero Thank you for your PR, just a small fix.

File: src/codec/postcard_1_0.rs (hunk 1 above)
Comment (lu-zero) [reply src/codec/postcard_1_0.rs]: Now hopefully should work

**Pull request merged**